wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java

```diff
@@ -144,8 +146,10 @@ private String htmlToxhtmlAndBase64Images(Page page) throws WikiException {
           }
           if(imageAttachment != null && imageAttachment.getMimeType() != null && imageAttachment.getMimeType().startsWith("image/")){
             byte[] bytes = imageAttachment.getContent();
+            ByteArrayInputStream buffArray = new ByteArrayInputStream(bytes);
+            BufferedImage buffImage = ImageIO.read(buffArray);
             element.attr("src", "base64," + Base64.encodeBase64String(bytes));
-            element.attr("style","width:100%;height:100%");
+            element.attr("style", "width:" + buffImage.getWidth() + "px;height:" + buffImage.getHeight() + "px;");
```

Comment:
Could you try/catch the exception inside this function and then when catching it you put default values for Height/Width

Reply:
Done